### DISCUSSION TITLE
Is there a way to set up different lualine configurations for different file types?

### DISCUSSION THREAD

**Discussion by LaBatata101** [nvim-lualine/lualine.nvim#567] [Q&A] [answered]
I'm using [dap](https://github.com/mfussenegger/nvim-dap) for debugging and I want to have a different status line look for the dap's buffers. Is this possible?
Each dap's buffer have a different file type.

![image](https://user-images.githubusercontent.com/20308796/152036502-705a6958-cb99-4faa-81d1-f4208d826461.png)

**Comment by fitrh** [comment, marked as answer]
You can create custom extension and set the `filetypes` key to the filetype of the buffer, here is an example of custom extension: [help extension](https://github.com/fitrh/init.nvim/blob/main/lua/plugin/lualine/extension/help.lua)

**Reply by LaBatata101** [reply]
Thanks!
